charts/home-assistant/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 2020.12.1
 description: Home Assistant
 name: home-assistant
-version: 5.2.1
+version: 5.3.1
```

Comment:
this is not a feature, please bump it to 5.2.2

Reply:
Done.